salesagility/SuiteCRM
# Fixed - #2075 Email templates chooser: sort by name!

## Description
Fixed - #2075  Email templates chooser: sort by name! 
Sorts email templates by name when selecting them in workflow.




## Motivation and Context
If you have a lot of templates it is very hand to find the one you are looking for.


## How To Test This
Create a workflow, choose action send mail, and then from the template dropdown see the list sorted by name.


## Types of changes


### Final checklist
